doc/config.rst

```diff
@@ -507,130 +517,138 @@ Stamp Type: LSST_Silicon
     * ``disable_field_rotation`` = *bool_value* (default = False) Do not consider the effect of the rotation of the camera relative to the spiders of the telescope during the exposure.
 
+
 Output types
 ============
 
-The output field is used to specify where to write output files and what format they should be.  There are several possibilities, including FITS files before and after electronics readout, and various types of truth information.  *imSim* adds the ``LSST_CCD`` type. It  understands how to write "eimage" files which are true representations of the electrons in the CCD including signals from the objects and cosmic rays with important physics effects such as the brighter-fatter effect and tree-rings applied.
+The output field is used to specify where to write output files and what format they should be.  There are several possibilities, including FITS files before and after electronics readout, and various types of truth information.  *imSim* adds the ``LSST_CCD`` type, an extra electronics ``readout`` type, and an ``opd`` type for writing out optical path difference images.
+
+*imSim*  understands how to write "eimage" files which are true representations of the electrons in the CCD including signals from the objects and cosmic rays with important physics effects such as the brighter-fatter effect and tree-rings applied.
 
 It can also write "amp" files. These are fully readout electronics files with one amplifier per FITS HDU with all of the proper headers needed to be processed by the Rubin Science Pipelines.  Both of these output formats can be examined with standard tools such as *ds9*.
 
-There are also several extra outputs available to the user including a centroid file containing the true position of the rendered sources, a list of optical path differences in the optical system, and a map of surface figure errors.
+There are also several extra outputs available to the user including a centroid file containing the true position of the rendered sources generated with the standard *GalSim* "truth" extra output type, a list of optical path differences in the optical system, and a map of surface figure errors.
 
 LSST CCD Sensor output and readout
 ----------------------------------
 
 Output Type: LSST_CCD
 ^^^^^^^^^^^^^^^^^^^^^
 
-For reading out LSST CCDs.
+For modifying the ``LSST_Image`` by adding cosmic rays and relevant headers and optionally writing out the "eimage" FITS files via the standard *GalSim* output.
 
 Optional keywords to set:
 """"""""""""""""""""""""""
 
     * ``cosmic_ray_rate`` = *float_value* (default = 0) The rate of cosmic rays per second in a sensor.
     * ``cosmic_ray_catalog`` = *str_value* (default = Distributed with *imSim*) A file containing cosmic ray images to paint on the sensor.
-    * ``header`` = *dictionary* (default = None) Extra items to add to the FITS header in the output files.
+    * ``header`` = *dict* (optional) Extra items to add to the FITS header in the output files.
```

Comment:
Should add here a few more, which aren't so obvious in the code, since we just use the standard GalSIm processing to handle them:

`nproc`: How many processes to use for working on multiple CCDs at a time.
`nfiles`: How many CCDs to build.  ("files" here refers to the eimage files that are produced by the standard processing)
`file_name`: The name of the eimage file to write out.
`dir`: The directory to write the output files to (default is the current directory)

Then there are also a couple imSim parameters you missed:

`camera` (default = 'LsstCam'), `exptime` (default=30), `det_num` (I think required currently, although the config files says it defaults to all 189, but I don't see that implemented anywhere).